### ISSUE TITLE
Symmetry relaxation works with new refinements, but not if continuing a refinement.

### ISSUE THREAD

**Issue by olibclarke** [3dem/relion#1023]
Hi,

Symmetry relaxation (--relax_sym) works if I start a fresh 3D-auto-refine job in relion 5-beta, but if I continue that same refinement (e.g. after replacing the mask), it gives the error "--relax_sym is not a valid relion argument". The continued refinement runs, but does not perform symmetry relaxation (whereas the fresh one does).

Cheers
Oli

**EDIT:** Just realized I had already reported this (#963 ) - sorry!

**Comment by biochem-fan** [comment]
Yes, this is a known issue. Because #963 remains open, please discuss there.
